docs/compiler-options.md

```diff
@@ -1,17 +1,17 @@
 # Options
 
-|  flag          | Default  | What it does? |
-|----------------|----------|---------------|
-| `-Oall`        | Disabled | Enables all compiler passes |
-| `-Ono-all`     | Disabled | Disables all compiler passes |
-| `-Oeta` `-Ono-eta` | Disabled | [eta-reduction](#eta-reduction) |
-| `-Oprune` `-Ono-prune` | Disabled | [definition-pruning](#definition-pruning) |
-| `-Olinearize-matches` `-Olinearize-matches-alt` `-Ono-linearize-matches` | Enabled  | [linearize-matches](#linearize-matches) |
-| `-Ofloat_combinators` `-Ono-float_combinators` | Enabled  | [float-combinators](#float-combinators) |
-| `-Omerge` `-Ono-merge` | Disabled | [definition-merging](#definition-merging) |
-| `-Oinline` `-Ono-inline` | Disabled | [inline](#inline) |
-| `-Ocheck-net-size` `-Ono-check-net-size` | Disabled | [check-net-size](#check-net-size) |
-| `-Oadt-scott` `-Oadt-num-scott` | adt-num-scott | [adt-encoding](#adt-encoding) | | 
+| flag                                                                     | Default       | What it does?                             |
+| ------------------------------------------------------------------------ | ------------- | ----------------------------------------- | --- |
```

Comment:
I think the extra |---| is breaking the table. If you look at the rich diff, you'll see it

Reply:
I think it is fixed now